.github/workflows/ci.yml

```diff
@@ -209,6 +212,22 @@ jobs:
           if [[ $GITHUB_REPOSITORY =~ /(silverstripe-spellcheck|recipe-authoring-tools)$ ]] || [[ "${{ matrix.phpunit_suite }}" == "recipe-authoring-tools" ]]; then
             sudo apt install -y hunspell libhunspell-dev hunspell-en-us
           fi
+          if [[ $GITHUB_REPOSITORY =~ /(silverstripe-dynamodb)$ ]]; then
+            # install dynamodb
+            mkdir _dynamodb
+            cd _dynamodb
+            sudo wget https://s3.ap-southeast-1.amazonaws.com/dynamodb-local-singapore/dynamodb_local_latest.tar.gz
+            sudo tar -xf dynamodb_local_latest.tar.gz
+            # run dynamo db as background process
+            # github actions comes java - `java -version` = openjdk version "11.0.18" 2023-01-17
```

Comment:
What does "github actions comes java" mean and why did you include a java command in this comment?

Reply:
Means java is installed out of the box, it's not a vanilla install of ubuntu.  Will update comment